sdk/src/Core/Amazon.Runtime/Internal/Util/SdkCache.cs

```diff
@@ -507,14 +507,14 @@ private set
             public CacheItem(T value)
             {
                 Value = value;
-                LastUseTime = GetCorrectedLocalTime();
+                LastUseTime = GetCorrectedTime();
             }
         }
 
-        private static DateTime GetCorrectedLocalTime()
+        private static DateTime GetCorrectedTime()
         {
 #pragma warning disable CS0612,CS0618 // Type or member is obsolete
-            return AWSSDKUtils.CorrectedUtcNow.ToLocalTime();
+            return AWSSDKUtils.CorrectedUtcNow;
```

Comment:
Now I'm wondering, is there even a point of having this method if we can just call AWSSDKUtils.CorrectedUtcNow()? Seems redundant

Reply:
No point in the private method anymore. I removed it changed all the places it was called to directly call `AWSSDKUtils.CorrectedUtcNow` in the next commit.